src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1181,6 +1212,47 @@ private void addNewFolder() {
 		}
 	}
 
+	private void renameFolder() {
+		renameFolder.setEnabled(true);

```

Comment:
Why this?

Reply:
may be something i forgot to remove while doing testings